pkg/apis/kops/validation/validation.go

```diff
@@ -85,17 +85,23 @@ func validateClusterSpec(spec *kops.ClusterSpec, c *kops.Cluster, fieldPath *fie
 
 	// SSHAccess
 	for i, cidr := range spec.SSHAccess {
-		allErrs = append(allErrs, validateCIDR(cidr, fieldPath.Child("sshAccess").Index(i))...)
+		if !strings.HasPrefix(cidr, "pl-") {
```

Comment:
Should we (can we) add validation that the PrefixListID is actually valid by interrogating AWS, or don't we do that during validation?

Reply:
Shouldn't this only be allowed if the cloudprovider is AWS?